capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative('decorator')
```

Comment:
[OPTIONAL] Kindly implement MIT license section to your readme and add MIT license to your project.